app/views/comparisons/change_in_solar_pv_since_last_year/_table.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= component 'comparison_table',
+              report: report, advice_page: advice_page, table_name: table_name, index_params: index_params,
+              headers: headers, colgroups: colgroups, advice_page_tab: advice_page_tab do |c| %>
+  <% @results.each do |result| %>
+    <% c.with_row do |r| %>
+      <% r.with_school school: result.school %>
+
+      <%= r.with_var val: result.previous_year_solar_pv_kwh, unit: :kwh %>
```

Comment:
Not that this really matters but don't think these need to be `<%=`, just `<%` should work.